protocols/1501-pennington-biomedical/BCA_assay.ot2.py

```diff
@@ -0,0 +1,74 @@
+from opentrons import labware, instruments
+
+metadata = {
+    'protocolName': 'BCA Assay',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# labware
+tube_rack_1 = labware.load('tube-rack-2ml', slot='3')
+tube_rack_2 = labware.load('tube-rack-2ml', slot='6')
+tip_rack_200 = [labware.load('tiprack-200ul', slot) for slot in ['7', '10']]
```

Comment:
Please change these tipracks to be separate per pipette model. Tip tracking prevents one pipette to have knowledge of another pipette's actions.